kuka_kr16_support/package.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<package>
+  <name>kuka_kr16_support</name>
+  <version>0.0.1</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR16 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 16 manipulators. This currently includes the KR 16-2 only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 16-2 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      found in the online
+      http://www.kuka-robotics.com/res/sps/e6c77545-9030-49b1-93f5-4d17c92173aa_Spez_KR_16_en.pdf
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Lars Tingelstad</author>
+  <maintainer email="lars.tingelstad@ntnu.no">Lars Tingelstad</maintainer>
+  <license>BSD</license>
+
+  <url type="website">http://ros.org/wiki/kuka_kr16_support</url>
+  <url type="bugtracker">https://github.com/ros-industrial/kuka_experimental/issues</url>
+  <url type="repository">https://github.com/ros-industrial/kuka_experimental</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>roslaunch</build_depend>
+
+  <run_depend>robot_state_publisher</run_depend>
+  <run_depend>joint_state_publisher</run_depend>
+  <run_depend>rviz</run_depend>
+  <run_depend>xacro</run_depend>
+
+  <export>
+    <architecture_independent/>
+  </export>
+</package>
```

Comment:
Seeing as this is a new package, and the KUKA pkgs are most likely to be released into Indigo, perhaps it would make sense to immediately start using *package manifest format 2*. See [REP-140](http://www.ros.org/reps/rep-0140.html) and [catkin 0.6.14 documentation » How to do common tasks » Package format 2 (recommended) » Migrating from format 1 to format 2](http://docs.ros.org/indigo/api/catkin/html/howto/format2/migrating_from_format_1.html#migrating-from-format1-to-format2).

I recently migrated all of the Fanuc pkgs to format 2. Perhaps the manifest of [fanuc_lrmate200ic_support](https://github.com/gavanderhoorn/fanuc/commit/0668d8ea72df0ac904139a130f57e9bc60b8b1a1#diff-badd9cd8169639ecf6863614d090b1b2) can serve as an example.